Investigate Fatal SIGILL Error

### Description
Investigate the fatal SIGILL error encountered in our acceptance tests [here](https://app.circleci.com/pipelines/github/ConsenSys/teku/10697/workflows/e67bb580-8643-4266-8d19-99d94af8698b/jobs/67884): 
```
RemoteValidatorAcceptanceTest > shouldCreateAttestationsWithRemoteValidator() FAILED
    org.testcontainers.containers.ContainerLaunchException at RemoteValidatorAcceptanceTest.java:48
        Caused by: org.rnorth.ducttape.RetryCountExceededException at RemoteValidatorAcceptanceTest.java:48
            Caused by: org.testcontainers.containers.ContainerLaunchException at RemoteValidatorAcceptanceTest.java:48
                Caused by: java.lang.IllegalStateException at RemoteValidatorAcceptanceTest.java:48

RemoteValidatorAcceptanceTest > shouldCreateAttestationsWithRemoteValidatorStartingFirst() FAILED
    java.lang.AssertionError at RemoteValidatorAcceptanceTest.java:59
        Caused by: org.awaitility.core.ConditionTimeoutException at RemoteValidatorAcceptanceTest.java:59
            Caused by: java.lang.AssertionError at Node.java:67
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGILL (0x4) at pc=0x00007f8a4ecec017, pid=8951, tid=8975
#
# JRE version: OpenJDK Runtime Environment (11.0.8+10) (build 11.0.8+10-post-Ubuntu-0ubuntu120.04)
# Java VM: OpenJDK 64-Bit Server VM (11.0.8+10-post-Ubuntu-0ubuntu120.04, mixed mode, sharing, tiered, compressed oops, g1 gc, linux-amd64)
# Problematic frame:
# C  [libblst.so+0x20017]
#
# Core dump will be written. Default location: Core dumps may be processed with "/usr/share/apport/apport %p %s %c %d %P %E" (or dumping to /home/circleci/project/acceptance-tests/core.8951)
#
# An error report file with more information is saved as:
# /home/circleci/project/acceptance-tests/hs_err_pid8951.log
#
# If you would like to submit a bug report, please visit:
#   https://bugs.launchpad.net/ubuntu/+source/openjdk-lts
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
```

--- a/gradle/versions.gradle
+++ b/gradle/versions.gradle
@@ -39,7 +39,7 @@ dependencyManagement {
     dependency 'io.protostuff:protostuff-runtime:1.6.2'
 
     dependency 'io.libp2p:jvm-libp2p-minimal:0.8.0-RELEASE'
-    dependency 'tech.pegasys:jblst:0.3.3'
+    dependency 'tech.pegasys:jblst:0.3.3-1'
 
     dependency 'org.hdrhistogram:HdrHistogram:2.1.12'
 

